README.md

```diff
@@ -34,10 +35,108 @@ Installation
 ember install ember-sinon-qunit
 ```
 
-
 Usage
 ------------------------------------------------------------------------------
 
+Ember-Sinon-QUnit supports two different versions of the Ember-QUnit API:
+
+1. The new QUnit hooks API, which takes a `hooks` object then wires up sandbox 
+creation and restoration to `beforeEach` and `afterEach` of the module, respectively.
+1. The classic API, which automatically wires up sandbox creation and restoration 
+to `QUnit.testStart` and `QUnit.testDone` respectively.
+
+### QUnit `hooks` API
+
+To use, import the setup method from within your test file and execute it.
+
+```js
+import { setupSinon } from 'ember-sinon-qunit/test-support';
```

Comment:
The name `setupSinon` is fairly general. Would it be easier for people to understand what this is doing by naming it `setupSinonRestoration`? That way they wouldn't even have to look at the `setupSinon` implementation in order to understand its intent.